Frameworks/jdi-matchers/src/main/java/com/epam/matcher/base/IChecker.java

```diff
@@ -13,128 +13,601 @@
 import static com.epam.jdi.tools.ReflectionUtils.getFields;
 import static java.util.Arrays.asList;
 
+/**
+ * Interface declares methods that are used for checking (contains, equals, empty...)
+ */
 public interface IChecker {
+    /**
+     * Returns checker with ignoreCase = true
+     *
```

Comment:
Are these empty lines really necessary? It won't affect jDoc formatting.
I believe it might be useful just to add "< p >"(without spaces)